epoch1/cuda/ee_mumu/SubProcesses/Makefile

```diff
@@ -73,6 +73,9 @@ else
   override RNDGEN = common
 endif
 
+# Set the host compiler for nvcc
+override NVCC := $(NVCC) -ccbin $(subst ccache ,,$(CXX))
+
```

Comment:
I think I understand now:
This force-overrides the `NVCC` variable. In the C++-only CI builds, it's empty, but after this line, it's not.

The subsequent parts test this if this variable is empty, however. If empty, no cuda is built. If set, it tries to build CUDA, but it expands to `"" -ccbin <expansion of CXX>`.